cli/command/context/show_test.go

```diff
@@ -0,0 +1,19 @@
+package context
+
+import (
+	"testing"
+
+	"gotest.tools/v3/assert"
+	"gotest.tools/v3/golden"
+)
+
+func TestShow(t *testing.T) {
+	cli := makeFakeCli(t)
+	createTestContext(t, cli, "current")
+	cli.SetCurrentContext("current")
+
+	cli.OutBuffer().Reset()
+	assert.NilError(t, runShow(cli))
+	expected := string(golden.Get(t, "show.golden"))
+	assert.Equal(t, cli.OutBuffer().String(), expected)
```

Comment:
In other places we use this (I _think_ using that approach also allows re-generating the golden files);

```suggestion
	golden.Assert(t, cli.OutBuffer().String(), "show.golden")
```

That said, I wonder if using a `golden` file for this is a bit too much, we can probably just check for a string;

```suggestion
	assert.Equal(t, cli.OutBuffer().String(), "current")
```

Reply:
I tried some variations; do you think we should also have some tests for when an override is used? Or are such cases already covered enough?)


```bash
$ docker context show
default

$ docker context create current
current
Successfully created context "current"

$ docker context use current
current
Current context is now "current"

$ docker context show
current

$ DOCKER_CONTEXT=default docker context show
default

$ docker --context=default context show
default
```

Reply:
testing DOCKER_CONTEXT / --context would actually test `dockerCli.CurrentContext()`, not the `show` command, and would require an actual e2e test, not a unit test relying on `fakecli`

Reply:
use of a golden file here is bit over-engineering, but I prefer this is aligned with other tests checking console output + it makes it easier to manage empty line at console end (otherwise we need `assert.Equal(t, out, "current\n")` which is not nice to read imho)